roles/openshift_provisioning_aws_vpc/molecule/default/playbook.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
+  become: no
+  pre_tasks:
+  - name: Start moto service
+    shell: nohup moto_server ec2 </dev/null >/dev/null 2>&1 &
+    changed_when: false
+  - name: Write boto config
+    copy:
+      src: boto_config
+      dest: /root/.boto
```

Comment:
@cooktheryan The pre_tasks here configure the moto ec2 service, and write a boto config that uses the service.